packages/peregrine/lib/store/reducers/cart.js

```diff
@@ -74,10 +76,7 @@ const reducerMap = {
             isAddingItem: false
         };
     },
-    [actions.updateItem.request]: (state, { payload, error }) => {
-        if (error) {
-            return initialState;
```

Comment:
We should not be passing error to the `request` sync actions, only the `receive`.